packages/react-components/src/components/Picker/Picker.tsx

```diff
@@ -204,6 +206,7 @@ export const Picker: React.FC<IPickerProps> = ({
           size={size}
           onTrigger={handleTrigger}
           onClear={handleClear}
+          dataTestId={dataTestId}
```

Comment:
I think we can simply use `data-testid` here and spread additional props in `Trigger` component.